- Don't overwrite package.cpath to make LUA_CPATH work.
- Exit if modules cannot be loaded.

diff --git a/tests/test.lua b/tests/test.lua
index 503e3f4..7168bbd 100755
--- a/tests/test.lua
+++ b/tests/test.lua
@@ -45,7 +45,7 @@ end
 local self = arg[0]
 
 package.path = "../?.lua;../?/init.lua"
-package.cpath = "./?-"..V..".so;./?-"..V..".dll;./?.so;./?.dll"
+package.cpath = "./?-"..V..".so;./?-"..V..".dll;./?.so;./?.dll;"..package.cpath
 if mode == "module" then
   print("testing Lua API using `compat53.module` ...")
   _ENV = require("compat53.module")
@@ -281,6 +281,7 @@ if utf8.len then
   ___''
 else
   print("XXX: utf8 module not available")
+  os.exit(1)
 end
 
 
@@ -291,6 +292,7 @@ if string.pack then
   ___''
 else
   print("XXX: string packing not available")
+  os.exit(1)
 end
 
 
